src/Microsoft.DotNet.XHarness.Apple/AppOperations/AppTester.cs

```diff
@@ -361,11 +361,15 @@ private async Task RunDeviceTests(
             // We need to check for MT1111 (which means that mlaunch won't wait for the app to exit)
             IFileBackedLog aggregatedLog = Log.CreateReadableAggregatedLog(_mainLog, testReporter.CallbackLog);
 
+            // The app output log is not directly accessible. 
+            // Hence, we duplicate the log to the main console log to simplify the UX of failure investigation
+            IFileBackedLog aggregatedLogAppOutput = Log.CreateReadableAggregatedLog(_mainLog, appOutputLog);
```

Comment:
Does it make sense to keep the `appOutputLog` as a separate log file if its content is duplicated to the main log (console log)? We might scratch it out completely, but it would be a bigger change to xharness then.

Reply:
I would keep the file for now. 
I don't think it is that expensive to keep it around and not sure if there is a dependency on its existence (I might be wrong). 
As for the changes, I think they are looking good.

Reply:
I'd remove it eventually, it's a bit confusing having two files with the same data.